.gitignore

```diff
@@ -1,8 +1,2 @@
-*.pyc
-pythia.wsgi
-.DS_Store
-._*
-log
-*.swp
-
-python_modules
+post_json.py
+*__pycache__*
```

Comment:
what happened to all the other stuff in .gitignore like *.pyc? Seems to have been removed incorrectly..